src/user-event/event-builder/common.ts

```diff
@@ -45,4 +45,14 @@ export const CommonEventBuilder = {
       },
     };
   },
+
+  press: () => {
+    return {
+      persist: jest.fn(),
+      nativeEvent: {
+        timestamp: new Date().getTime(),
```

Comment:
Perhaps we could provide some `setEventTimestamp` function to inject timestamp value or function generating timestamp values so that we could have easier time working with snapshots? Wdyt?

Reply:
The idea behind this function is non-exported test util for our own convenience, as we are potentially interested in comparing event sequence with snapshot, etc. The users will probably not need it, unless they are doing some timing calculation, but in such case  they will just mock `Date.now()` etc. 

There are couple of ways we could achieve similar result:
1. Event timestamp setter function:
```
let getEventTimestamp = () => Date.now();
function setEventTimestamp(fn: () => Date) {
  getEventTimestamp = fn
}
```

2. Spying on event timestamp getter
```
function getEventTimestamp() {
  return Date.now();
}

jest.spyOn(UserEvent, "getEventTimestamp").mockImplementation(() => 0);
```

This option is to mock `Date.now()` in our tests, but that feels fragile compared to other approaches.

Reply:
I don't think this approach could work. The issue is that we wait for the press duration between calling `onResponderGrant` and `onResponderRelease` and RN will call onLongPress or onPress based on the time difference between those two events. If we simply mock the date, then the timestamp will stay the same and longPress will not work anymore. What I would suggest is to write a test with fake timers enabled where we use `jest.setSystemTime` to check for the events payload and skip that test for real timers. Either that or write the same test and do an inline snapshot of events with timestamp set to 0 but I don't think there would be much value in that